storm-core/src/jvm/backtype/storm/utils/RateTracker.java

```diff
@@ -87,7 +95,7 @@ public final float reportRate() {
             sum += _histograms[i];
         }
 
-        return sum / (float) duration * 1000;
```

Comment:
Isn't the disruptor queue using RateTracker already?  Is this a bug in rate tracker where it said it was doing milliseconds but was really doing seconds?  Or is this going to make the disruptor metrics wrong?